doc/overview.rst

```diff
@@ -5,10 +5,11 @@ Overview
 Introduction
 ============
 
-Dune is a build system for OCaml and Reason. It is not intended as a
-completely generic build system that is able to build any given project
-in any language. On the contrary, it makes lots of choices in order to
-encourage a consistent development style.
+Dune is a build system for OCaml and Reason (with Coq support in
+progress).  It is not intended as a completely generic build system
```

Comment:
I would change the first sentence to say : 
```
Dune is a build system for OCaml (with support for Reason and Coq).
```
Am not sure it is necessary to mention that Coq support is in progress; the exact level of support is explained in the stanza documentation.

Reply:
Agreed with @nojb on the opening statement.

Reply:
And it'll age better :)

Reply:
Thanks for the suggestions :grinning: